pandas/tests/test_sorting.py

```diff
@@ -413,10 +413,13 @@ def test_mixed_integer_from_list(self):
     def test_unsortable(self):
         # GH 13714
         arr = np.array([1, 2, datetime.now(), 0, 3], dtype=object)
-        msg = (r"'(<|>)' not supported between instances of ('"
-               r"datetime\.datetime' and 'int'|'int' and 'datetime\.datetime"
-               r"')|"
-               r"unorderable types: int\(\) > datetime\.datetime\(\)")
+        msg = ("unorderable types: ({0} [<>] {1}|{1} [<>] {0})".format(
+                   r"int\(\)", r"datetime\.datetime\(\)")  # noqa: E126
```

Comment:
this fails the wheel building CI. pls don't revert this, I don't want to debug this problem. Just make this MUCH simpler so it will pretty much always pass.

Reply:
It will *not* fail the wheel building CI. I actually checked the wheel builder failures from #25867 before giving my [review](https://github.com/pandas-dev/pandas/pull/25874#pullrequestreview-218710820) in #25874.

In case you don't have time to read the review, the original code in #25752 has an `if _np_version_under1p14 else`-switch that turned out to be wrong, while the regex here only used a `|`-conjunction.

> [...] I don't want to debug this problem.

Neither did I, but I've had to for a passing CI. Please have a look at the actual regexes. What I've written is a proper superset of the current regexes (again, now that the switch has been turned into `"|"`).

Reply:
> Just make this MUCH simpler so it will pretty much always pass.

Done.